spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/PubsubAdmin.java

```diff
@@ -194,10 +194,18 @@ public void deleteSubscription(String subscriptionName) {
 		return Lists.newArrayList(subscriptionsPage.iterateAll());
 	}
 
+	/**
+	 * @return the default acknowledgement deadline value in seconds
+	 */
 	public int getDefaultAckDeadline() {
 		return this.defaultAckDeadline;
 	}
 
+	/**
+	 * Sets the default acknowledgement deadline value.
```

Comment:
Nit-pick.
Method JavaDocs should be *imperative* not descriptive.
In other words: `Set the default ....`